benchmarks/README.md

```diff
@@ -0,0 +1,10 @@
+## Benchmarks
+
+This section contains some benchmarking results of numalogic's algorithms on real as well
+synthetic data. Datasets here are publicly available from their respective repositories.
+
+Note that efforts have not really been made on hyperparameter tuning. This is just to give users an 
+idea on how each algorithm is suitable for different kinds of data, and shows how they can do
```

Comment:
Overall LGTM. But, I think there should be a more detailed Readme explaining things in simpler terms so that even non ML engineers could understand it.

Reply:
For sure. Slowly we can add more details here.